[Functions] Send the placeholder appcheck token in case of error (#6750)
firebase/firebase-android-sdk: firebase-functions/CHANGELOG.md

# Unreleased

> * [fixed] Fixed an issue that prevented the App Check token to be handled correctly in case of error.

Reviewer: ```suggestion
* [fixed] Fixed an issue that prevented the App Check token from being handled correctly in case of error.
```
